server/build.gradle

```diff
@@ -34,6 +34,7 @@ dependencies {
   api project(":libs:elasticsearch-plugin-api")
   api project(":libs:elasticsearch-plugin-analysis-api")
   api project(':libs:elasticsearch-grok')
+  api project(":libs:elasticsearch-tdigest")
```

Comment:
Can we remove the official tdigest dependency here?

Reply:
Done.